src/CTA.Rules.PortCore/PortCoreRulesCli.cs

```diff
@@ -12,8 +12,8 @@ public class PortCoreOptions
         [Option('s', "solution-path", Required = false, HelpText = "Solution file path.")]
         public string SolutionPath { get; set; }
 
-        [Option('r', "rules-input-infile", Required = false, HelpText = "Rules json input file")]
-        public string RulesInputFile { get; set; }
+        [Option('r', "rules-input-infile-directory", Required = false, HelpText = "Directory containing rules json input file(s)")]
+        public string RulesInputFilesDirectory { get; set; }
```

Comment:
nit: We can shorten the names to rules-dir and RulesDir to stay consistent with the other variables.

Reply:
Good suggestion, will do.